packages/grafana-ui/src/components/Logs/LogRowMessage.tsx

```diff
@@ -110,7 +112,7 @@ class UnThemedLogRowMessage extends PureComponent<Props> {
             />
           )}
           <span className={cx(styles.positionRelative, { [styles.rowWithContext]: contextIsOpen })}>
-            {needsHighlighter ? (
+            {needsHighlighter && entry.length < MAX_CHARACTERS ? (
```

Comment:
`entry.length < MAX_CHARACTERS` could probably just be added to the `needsHighlighter` definition
Though that definition itself could use some simplification, so maybe something like
`const needsHighlighter = !!highlights?.[0]?.length && entry.length < MAX_CHARACTERS;`

Reply:
Yeah, great point!